app/commands/user/acquired_badge/create.rb

```diff
@@ -3,7 +3,11 @@ class AcquiredBadge
     class Create
       include Mandate
 
-      initialize_with :user, :slug
+      def initialize(user, slug, skip_email: false)
+        @user = user
+        @slug = slug
+        @skip_email = skip_email
+      end
```

Comment:
We really need to support `initialize_with` in mandate for kwargs!

Reply:
We do!